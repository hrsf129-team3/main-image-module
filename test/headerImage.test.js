import React from 'react';
import HeaderImage from '../client/src/components/HeaderImage.jsx';

describe('HeaderImage', () => {
  it('should render header image component', () => {
    const component = shallow(<HeaderImage />);
    expect(component).toMatchSnapshot();
  });
})


