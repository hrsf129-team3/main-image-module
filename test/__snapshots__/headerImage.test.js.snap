// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderImage should render header image component 1`] = `ShallowWrapper {}`;
